EATSSU/App/Sources/Data/Firebase/LaunchSourceManager.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  LaunchSourceManager.swift
+//  EATSSU
+//
+//  Created by 황상환 on 3/23/25.
+//
+
+import Foundation
+import UIKit
+import FirebaseAnalytics
+
+enum LaunchSource: String {
+    case icon = "icon"
+    case localNotification = "local_notification"
+    case widget = "widget"
+    case background = "background"
+    
+}
```

Comment:
케이스명과 rawValue값 같다면 생략도 가능합니다

```suggestion
enum LaunchSource: String {
    case icon
    case localNotification = "local_notification"
    case widget
    case background
}
```